src/coreclr/debug/daccess/request.cpp

```diff
@@ -2577,7 +2577,7 @@ ClrDataAccess::GetAssemblyData(CLRDATA_ADDRESS cdBaseDomainPtr, CLRDATA_ADDRESS
     }
 
     assemblyData->AssemblyPtr = HOST_CDADDR(pAssembly);
-    assemblyData->ClassLoader = HOST_CDADDR(pAssembly->GetLoader());
+    assemblyData->ClassLoader = 0;
```

Comment:
This change LGTM. Are you going to make a matching change in the SOS to delete ClassLoader from the DumpAssembly output?

Reply:
I thought I would leave it in SOS for now.  It will still print something valid for older runtimes and 0 for .NET 9.  But if you insist, I'll remove it.

Reply:
I think it would be an improvement to delete it from SOS output. The information that it prints for older runtimes is not very useful.